sys/conf/kern.mk

```diff
@@ -217,7 +217,7 @@ CFLAGS+=	-fstack-protector
 # but critical for these newer versions.
 #
 .if ${CFLAGS:M-g} != "" && ${CFLAGS:M-gdwarf*} == ""
-CFLAGS+=	-gdwarf-4 -ggdb3
+CFLAGS+=	-gdwarf-2
```

Comment:
Probably but that is a HEAD issue, this reverts the tree back to HEAD state.  Newer GCC would be fine with this too so that may cause extra work but
CFLAGS.clang+= and CFLAGS.gcc+= could be proposed